website/content/docs/concepts/cpu.mdx

```diff
@@ -143,19 +144,18 @@ this option restricts tasks from bursting above their CPU limit even when there
 is idle capacity on the node. The tradeoff is consistency versus utilization.
 A task with too few CPU resources may operate fine until another task is placed
 on the node causing a reduction in available CPU bandwidth, which could cause
-distruption for the underprovisioned task.
+disruption for the underprovisioned task.
 
 ### CPU Environment Variables
 
 To help tasks understand the resources available to them, Nomad sets the
 following environment variables in their runtime environment.
 
 
-- <code>NOMAD_CPU_LIMIT</code> - The amount of CPU bandwidth allocated on behalf of the
-task.
-
-- <code>NOMAD_CPU_CORES</code> - The set of cores in [cpuset][] notation reserved for the
```

Comment:
These `<code>` tags were somehow causing the remaining markdown (the link and backticks) in the line item to be unrendered. And the `<code>` tag also means there's no anchor, which we should have.